code/src/java/pcgen/cdom/content/KnownSpellIdentifier.java

```diff
@@ -146,28 +147,24 @@ public Integer getSpellLevel()
 		if (ref == null)
 		{
 			List<Spell> returnList = new ArrayList<>();
-			for (CDOMList<Spell> list : classSpellLists)
+			classSpellLists.forEach(list ->
 			{
 				returnList.addAll(pc.getSpellsIn(list, spellLevel));
-			}
+			});
 			return returnList;
 		}
-		List<Spell> spellList = new ArrayList<>();
-		for (Spell sp : ref.getContainedObjects())
+		Collection<Spell> spellList = new ArrayList<>();
+		ref.getContainedObjects().forEach((Spell sp) ->
```

Comment:
This is one where the name does not match the content. I suspect this was the result of a bad merge. Closing.